paper/basic_training.tex

```diff
@@ -616,28 +620,52 @@ \subsubsection{Popular Thermostats}
+         The isokinetic (constant kinetic energy) ensemble samples the
+         same configurational phase space as the canonical ensemble, so
+         position-dependent equilibrium properties can be obtained
+         equivalently with either
+         ensemble\cite{Minary:2003:JChemPhys:Algorithms}.  However, the
+         isokinetic ensemble is not properly sampled by the simple
+         velocity rescaling thermostat, and its usage can lead to
+         simulation artifacts, so it is not
+         recommended\cite{Braun:2018:arXiv:Anomalous}.
 
     \item \textbf{Hoover (Gaussian)}
+        \todo[inline, color={green!20}]{EB: I recommend referring to
+        this thermostat as the Gaussian Thermostat. I've seen it
+        referred to as the Hoover-Evans thermostat as well (I
+        haven't seen it called the Hoover thermostat, but I'm sure
+        somebody has called it that), but simply calling it the
+        Gaussian thermostat avoids
+        wading into the argument about who came up with it
+        first.}
         
-        This thermostat is also an example of a velocity rescaling thermostat; however, the goal of the Gaussian thermostat is to ensure the change in
-        temperature $\Delta T$ is always 0 ($\Delta T \equiv 0$).
+        The goal of the Gaussian thermostat is to ensure the change in
+        instantaneous temperature $\Delta T$ is always 0 ($\Delta T \equiv 0$), which
+        is accomplished by modifying the force calculation with the form
+        $F = F_{interaction} + F_{Lagrange multiplier}$, where
+        $F_{interaction}$ is the standard interactions calculated during
+        the course of the simulation and $F_{Lagrange multiplier}$ keeps
+        the kinetic energy constant.
         The reasoning for the naming of this thermostat is due to the method to solve for the smallest perturbative forces needed to keep the change in temperature equal to 0.
         Using the Gaussian principle of least constraint, forces are calculated to maintain a net 0 change in temperature, while minimally perturbing the system\cite{thermostatAlgorithms2005}.
 
-        Important information to know about this thermostat is that it is reversible and canonical; however, there are no temperature
-        fluctuations.
-        This once again will lead to issues when trying to sample any properties from the canonical ensemble.
-        Also, due to the nature of the thermostat preventing a change in temperature, whatever temperature
-        the system is at when the thermostat is applied, it will stay there. 
+        This thermostat is reversible and it samples the isokinetic
+        ensemble. Position-dependent equilibrium properties will
+        therefore be in agreement with the canonical ensemble (as
+        explained above), but dynamical properties will not.  due
```

Comment:
"due" is lowercase at the start of a sentence, oops.

Reply:
Fixed.